### PR TITLE
NXO Demo: add patch order after shipping change

### PR DESCRIPTION
**Pull request by jcnoriega**
Thank you for your contribution to PayPal. 

> Before submitting this PR, note that we cannot accept language translation PRs. We have a dedicated localization team to provide the translations. If there is an error in a specific translation, you may open an issue and we will escalate it to the localization team.

### Summary of changes

 - This PR patches an order after a change in shipping address

 ### Checklist

 - [ ] Added a changelog entry

-

### CODE REVIEW

File: Demo/src/main/java/com/paypal/android/viewmodels/PayPalNativeViewModel.kt [hunk 1]
```diff
@@ -75,7 +92,43 @@ class PayPalNativeViewModel @Inject constructor(
...
+                ),
+                PatchAmount.Replace(
+                    purchaseUnitReferenceId = "PUHF",
+                    amount = getAmount(
+                        value = order.purchaseUnitList.first().amount.value,
+                        shippingValue = updatedShippingAmount ?: "0.00"
+                    )
+                )
+            )
+            shippingChangeActions.patchOrder(patchRequest) {
```
Comment (scannillo) [inline Demo/src/main/java/com/paypal/android/viewmodels/PayPalNativeViewModel.kt:129]: What does it mean for a merchant to patch an order? Basically to amend an existing order?

Comment (jcnoriega) [reply Demo/src/main/java/com/paypal/android/viewmodels/PayPalNativeViewModel.kt:129]: I think so. Its for when a shipping address is changed or a shipping method. The order needs to be updated with the new value.

File: Demo/src/main/java/com/paypal/android/usecase/GetBillingAgreementTokenWithoutPurchaseUseCase.kt [hunk 2]
```diff
@@ -0,0 +1,52 @@
+package com.paypal.android.usecase
+
+import com.google.gson.JsonObject
+import com.google.gson.JsonParser
+import com.paypal.android.api.services.PayPalApi
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import javax.inject.Inject
+
+class GetBillingAgreementTokenWithoutPurchaseUseCase
```
Comment (scannillo) [inline Demo/src/main/java/com/paypal/android/usecase/GetBillingAgreementTokenWithoutPurchaseUseCase.kt:10]: What is a billing agreement without purchasing? Is this the same as vault without purchase?

**Comment by scannillo** [comment]
Is Northstar Android out for beta release like iOS? If so - this needs a CHANGELOG entry.

**Pull request closed without merging**